core-bundle/contao/drivers/DC_Table.php

```diff
@@ -246,6 +246,31 @@ public function __construct($strTable, $arrModule=array())
 		}
 	}
 
+	/**
+	 * Returns the current user changes merged with the database information.
+	 *
+	 * @return array<string, mixed>|null
+	 * @throws AccessDeniedException     if the current user has no read permission on the current record
+	 */
+	public function getActiveRecord(): array|null
+	{
+		$currentRecord = $this->getCurrentRecord();
+
+		if (null === $currentRecord)
+		{
+			return null;
+		}
+
+		return array_merge($currentRecord, array_map(static function ($value) {
+			if (\is_string($value) || \is_bool($value) || \is_float($value) || \is_int($value) || $value === null)
```

Comment:
I would add a comment to the database statement as where this code is originating from 🙈

Reply:
Added in efe1b875dce296708c6b266e012cfaa60e9f0218.